src/wallet/wallet.cpp

```diff
@@ -2570,6 +2828,16 @@ bool CWallet::CommitTransaction(CWalletTx &wtxNew, CReserveKey &reservekey)
     {
         if (fBroadcastTransactions)
         {
+            /** When the wallet is parallelized, this will higher performing, however right now its a wash.
+                Enqueuing like this will not provide feedback if the the mempool doesn't accept the tx.
```

Comment:
is this a placeholder chunk of code for when wallet will be fully parallelized?

Reply:
Yes, I've fully tested this code as well as the code below.  It works fine but it doesn't allow the CommitTransaction to return that the mempool has rejected the tx.  So that's a con.  And right now there is no "pro" -- although in theory it should increase wallet performance to push the mempool admission to happen asynchronously, in my testing it didn't matter much.